Update Intro CS course (#1292)
ossu/computer-science: coursepages/intro-cs/README.md

# Introduction to Computer Science and Programming using Python

> This course will introduce you to the world of computer science an programming. This course gives you a flavor of the material to come. If you finish the course wanting more, Computer Science is likely for you!

Reviewer: ```suggestion
This course will introduce you to the world of computer science and programming, giving you a flavor of the material to come. If you finish the course wanting more, Computer Science is likely for you!
```

This course has been developed by MIT and is available on [MIT's OCW](https://ocw.mit.ed) platform.

> Introduction to Computer Science and Programming in Python is intended for students with little or no programming experience. It aims to provide students with an understanding of the role computation can play in solving problems and to help students, regardless of their major, feel justifiably confident of their ability to write small programs that allow them to accomplish useful goals. The class uses the Python programming language.

**Course Link:** <https://ocw.mit.edu/courses/6-100l-introduction-to-cs-and-programming-using-python-fall-2022/pages/material-by-lecture/>

## ⚠️ Under Review

> This course is under review. While this course has been tried to be designed to accomodate people completely new to programming, if you find this course difficult to follow, you can try doing one of the [Intro to Programming courses](coursepages/intro-programming/README.md) first, and then do this course. We would like to hear your feedback regarding this course, and whether you needed to do the [Intro to Programming courses](coursepages/intro-programming/README.md) first. Please provide your feedback on this RFC: <https://github.com/ossu/computer-science/issues/1266>.

Reviewer: ```suggestion
This course is under review. The course has been designed to accommodate people completely new to programming. However, if you find the course difficult to follow, you can try doing one of the [Intro to Programming courses](coursepages/intro-programming/README.md) and then return to this course. We would like to hear your feedback regarding this course, and whether you needed to do the [Intro to Programming courses](coursepages/intro-programming/README.md) first. Please provide your feedback on this RFC: <https://github.com/ossu/computer-science/issues/1266>.
```
